src/proxy/build.gradle

```diff
@@ -39,9 +39,21 @@ jar.enabled = false
 build.dependsOn shadowJar
 buildall.dependsOn shadowJar
 
+task licenseFormatJava(type:nl.javadude.gradle.plugins.license.License) {
+    source = fileTree('src/main/java')
+}
+
+task licenseTestJava(type:nl.javadude.gradle.plugins.license.License) {
+    source = fileTree('src/main/java')
+    check = true
+}
+
 licenseFormatMain.enabled = false
 licenseMain.enabled = false
 
+licenseFormat.dependsOn licenseFormatJava
+licenseTest.dependsOn licenseTestJava
+
```

Comment:
Is it really necessary to modify this `build.gradle`?

Reply:
Without this change Gradle license plugin skipped `proxy` when running `licenseFormat`.  The same configuration is present in `build.gradle` of other modules, e.g. [signer](https://github.com/nordic-institute/X-Road/blob/develop/src/signer/build.gradle#L132). Afer this change proxy`s `build.gradle` configuration is aligned with other modules.

Reply:
Ok